keyboards/uno/v2/readme.md

```diff
@@ -0,0 +1,19 @@
+# uno

```

Comment:
Please use Unix line endings for all files.

Reply:
Ok, I think I did it. If not please elaborate